Widgets/RecentDonations.html

```diff
@@ -0,0 +1,35 @@
+<html>
+	<head>
+		<script type="text/javascript" src="../bower_components/jquery/dist/jquery.min.js"></script>
+		<script type="text/javascript" src="../bower_components/angular/angular.min.js"></script>
+		<script type="text/javascript" src="../js/Globals.js"></script>
+		<script type="text/javascript" src="../js/RecentDonations.js"></script>
+
+		<link href="http://fonts.googleapis.com/css?family=Cantarell:700" rel="stylesheet" type="text/css">
+
+		<link rel="stylesheet" type="text/css" href="widget.css">
+	</head>
+	<body>
+		<div id="widget" ng-app="RecentDonations" ng-controller="RecentDonationsControl">		
+
+			<h1>Recent Donations for {{ displayName }} </h1>
+
+			<div id="error" ng-show="error">{{ error }}</div>
+
+			<table ng-hide="error">
+		    <tr>
+				<th>Name</th>
+				<th>Amount</th>
+				<th>Message</th>
+				<th>Date</th>
+			</tr>
+			  <tr ng-repeat="x in contributors">
+			    <td>{{ x.name }}</td>
+			    <td>{{ x.amount }}</td>
+			    <td>{{ x.message }}</td>
```

Comment:
This was mostly just to see if I could display it. I think I'll simplify it; a list of top N name/amounts, and then display the quotes one at a time or something so that it's interesting